Tasks/ANT/ant2.js

```diff
@@ -13,6 +13,14 @@ antb.arg(tl.getPathInput('antBuildFile', true, true));
 antb.arg(tl.getDelimitedInput('options', ' ', false));
 antb.arg(tl.getDelimitedInput('targets', ' ', false));
 
+// update ANT_HOME if user specified path manually
+var antHomeUserInputPath = tl.getPathInput('antHomeUserInputPath', false, false);
```

Comment:
Shouldn't the second boolean argument to getPathInput() be 'true' so that the path is checked, matching the logic in ant.ps1?